src/test/java/io/github/thunderz99/cosmos/util/JsonUtilTest.java

```diff
@@ -1,73 +1,120 @@
-		assertThat(JsonUtil.toJson(data)).isEqualTo(String.format("{%n  \"id\" : 1%n}"));
-	}
+        Data data = JsonUtil.fromJson(json, this.getClass().getName() + "$Data");
+        assertThat(data.id).isEqualTo(3);
 
-	@Test
-	public void shouldBeConvertToJsonByClassName() {
+    }
 
-		var json = "{\"id\":3}";
+    @Test
+    public void shouldConvertToList() {
 
-		Data data = JsonUtil.fromJson(json, this.getClass().getName() + "$Data");
-		assertThat(data.id).isEqualTo(3);
+        var json = "[{\"id\":3}, {\"id\":5}]";
 
-	}
+        List<Data> data = JsonUtil.fromJson2List(json, this.getClass().getName() + "$Data");
+        assertThat(data.get(1).id).isEqualTo(5);
 
-	@Test
-	public void shouldConvertToList() {
+    }
 
-		var json = "[{\"id\":3}, {\"id\":5}]";
+    @Test
+    public void shouldConvertToListOfMap() {
 
-		List<Data> data = JsonUtil.fromJson2List(json, this.getClass().getName() + "$Data");
-		assertThat(data.get(1).id).isEqualTo(5);
+        var json = "[{\"id\":3}, {\"id\":5}]";
 
-	}
+        var data = JsonUtil.toListOfMap(json);
+        assertThat(data.get(1).get("id")).isEqualTo(5);
 
-	@Test
-	public void shouldConvertToListOfMap() {
+    }
 
-		var json = "[{\"id\":3}, {\"id\":5}]";
+    @Test
+    public void shouldBeConvertToNoIndentJson() {
+        var data = new Data(1);
+        assertThat(JsonUtil.toJsonNoIndent(data)).isEqualTo("{\"id\":1}");
+    }
 
-		var data = JsonUtil.toListOfMap(json);
-		assertThat(data.get(1).get("id")).isEqualTo(5);
+    @Test
+    public void shouldBeConvertFromJson() {
+        var json = "{\"id\":1}";
+        assertThat(JsonUtil.fromJson(json, Data.class).id).isEqualTo(1);
+    }
 
-	}
+    @Test
+    void shouldBeError() {
+        var json = "{\"id\":1, \"name\":}";
+        assertThatThrownBy(() -> JsonUtil.fromJson(json, Data.class))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("json process error.");
+    }
 
-	@Test
-	public void shouldBeConvertToNoIndentJson() {
-		var data = new Data(1);
-		assertThat(JsonUtil.toJsonNoIndent(data)).isEqualTo("{\"id\":1}");
-	}
+    @Test
+    void shouldBeEnumNullWithoutError() {
+        var json = "{\"id\":1,\"secondType\":\"INTEGER\",\"firstType\":\"STRING\",\"type\":\"STRING1\"}";
+        DataWithEnum dataWithEnum = JsonUtil.fromJson(json, DataWithEnum.class);
+        assertThat(dataWithEnum).isNotNull();
+        assertThat(dataWithEnum.id).isEqualTo(1);
+        assertThat(dataWithEnum.type).isEqualTo(null);
+        assertThat(dataWithEnum.firstType).isEqualTo(Type.STRING);
+        assertThat(dataWithEnum.secondType).isEqualTo(Type.STRING);
+    }
 
-	@Test
-	public void shouldBeConvertFromJson() {
-		var json = "{\"id\":1}";
-		assertThat(JsonUtil.fromJson(json, Data.class).id).isEqualTo(1);
-	}
+    @Test
+    void t() {
+        Throwable ex = new InvalidParameterException("invalid paramter");
+        System.out.println(JsonUtil.exceptionToString(ex));
```

Comment:
Please rename this method's name to a more readable and meaningful name that other developer can comprehend this test case.